app/src/main/java/com/example/bookwormadventuresdeluxe2/MyBooksFragment.java

```diff
@@ -0,0 +1,70 @@
+package com.example.bookwormadventuresdeluxe2;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link MyBooksFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class MyBooksFragment extends Fragment
```

Comment:
Since the three fragments are similar (MyBooksFragment, ProfileFragment, RequestsFragment), I think it makes sense to create a parent fragment class then have these 3 fragments extend them. Or if they are identical (hard to tell without cloning this) maybe we can just have a single class for all 3? If you foresee that these 3 will need to be different then the parent class makes more sense.

Reply:
Thanks for noticing that. The fragment classes have minimal functionality at the moment. 
As we start changing buttons and editTexts the onCreateView will be significantly different. 
We will likely be passing different objects to each fragment hence the other methods would be different.
Which leaves only the empty constructor to be similar. 
Should we create a superClass for them?

Reply:
If its only the empty constructor that will be the same it makes sense to leave it as is